### DISCUSSION TITLE
background opacity at top-bar?

### DISCUSSION THREAD

**Discussion by OmiChrome** [tomasklaen/uosc#356] [Q&A] [answered]
i want to change the opacity behind the `title` in top-bar 
![image](https://user-images.githubusercontent.com/108952834/198889600-33fd06ee-a4e8-42d6-b23b-21d008b8e38a.png)
 **look at the top-left corner** 🙏 

![image](https://user-images.githubusercontent.com/108952834/198889766-0b9cf8ba-5a89-4fc5-b983-68636cee973c.png)

**Comment by christoph-heinrich** [comment, marked as answer]
The background behind your title is too short. I used to have a similar problem, but setting `osd-font` in mpv.conf fixed it.

**Reply by OmiChrome** [reply]
but, whichi  `osd-font` should  i set 

```
osd-font-size=19
osd-font='Jetbrains Mono'                
```

this are my already set font  , and the image is taken from those

**Reply by christoph-heinrich** [reply]
That's weird because it doesn't look like a monospace font in the screenshot. Maybe you don't have `Jetbrains Mono` installed and so it falls back on something else? I'd generally recommend to not use use monospace fonts for user interfaces.
I'm currently using `Liberation Sans`. You'll have to choose one based on your installed fonts and your preferences.

**Reply by OmiChrome** [reply]
> `Jetbrains Mono` 
thx , for it , the issue is with `jetbrains fonts`  , also non mono font

**Reply by tomasklaen** [reply]
I think the background was too short because he's using `font_bold=yes`, and `text_width()` up until now was ignoring this and defaulting bold to `false`. Fixed in d9e5853.

And the option to control top bar background opacity is `top_bar_title_opacity=0.8`.
